.cirrus.yml

```diff
@@ -7,7 +7,7 @@ task:
     pkg install -y cmake libzmq4 ninja git
   build_script: |
     mkdir build && cd build
-    cmake -GNinja -DHELICS_BUILD_TESTS=ON -DHELICS_BUILD_EXAMPLES=ON ..
+    cmake -GNinja -DHELICS_BUILD_TESTS=ON -DHELICS_BUILD_EXAMPLES=ON -DCMAKE_CXX_STANDARD=17 ..
```

Comment:
Should the standard here be set to 20?

Reply:
I set it to 17 since there is a compile issue that I didn't want to deal with in this PR.   When I changed the default to 20, cirrus stopped building.   Needs to be fixed but didn't want to deal with that here.